.github/workflows/deploy-template.yml

```diff
@@ -31,16 +31,11 @@ jobs:
         run: |
           echo "triggered via schedule"
           echo "environments=${{inputs.auto_deploy_envs}}" >> $GITHUB_ENV
+
       - name: Set output
         id: set-environments
         run: |
           echo "environments=${{ env.environments }}" >> $GITHUB_OUTPUT
-  release:
-    needs: [prepare-values]
-    if: github.ref == 'refs/heads/master'
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v4
 
       - name: Configure AWS Credentials
```

Comment:
This step was in the `release` section, but now it's in the `prepare-values` section. Was that intentional or